dynawo/sources/API/DYD/DYDModelTemplateImpl.cpp

```diff
@@ -59,12 +59,30 @@ using boost::shared_ptr;
 namespace dynamicdata {
 
 ModelTemplate::Impl::Impl(const string& id) :
-Model::Impl(id, Model::MODEL_TEMPLATE) {
+Model::Impl(id, Model::MODEL_TEMPLATE),
+useAliasing_(true),
+generateCalculatedVariables_(true) {
 }
 
 ModelTemplate::Impl::~Impl() {
 }
 
+void
+ModelTemplate::Impl::setCompilationOptions(bool useAlias, bool generateCalculatedVariables) {
+  useAliasing_ = useAlias;
```

Comment:
It could be nice to use the same wording all the way long to ease later on researches for example (useAlias and useAliasing).

Reply:
Done